open_prime_rando/echoes/specific_area_patches.py

```diff
@@ -67,3 +70,27 @@ def torvus_generator(editor: PatcherEditor):
     obj = area.get_instance(2686994)
     obj.add_connection("Zero", "Increment", layer_cont1)
     obj.add_connection("Zero", "Increment", layer_cont2)
+
+
+def torvus_temple_crash(editor: PatcherEditor):
+    """
+    Remove cosmetic objects from Torvus Temple to minimize the chance of chance via alloc failure
+    """
+    world = editor.get_mlvl(TORVUS_BOG_MLVL)
+    area = world.get_area(TORVUS_TEMPLE_MREA)
+
+    default = area.get_layer("Default")
+    first_pass = area.get_layer("1st Pass")
+    second_pass = area.get_layer("2nd PAss")
+
+    for obj in ["Thrust1", "Thrust1", "Thrust2", "Thrust2", "Looping Thrust w/Doppler", "Looping Thrust w/Doppler"]:
+        first_pass.remove_instance(obj)
+
+    for obj in ["GENERATE GIBS"]:
+        second_pass.remove_instance(obj)
+
+    # These are generated objects
+    # debris = ["SwampCrateDebris"] * 7
+    # for obj in ["GibFlash", "Sound - Swamp Crate Gib"] + debris:
+    #     default.remove_instance(obj)
+
```

Comment:
?

Reply:
These are things I wanted to remove as well, but they're generated objects.

Reply:
gotcha. I assume to safely remove them we'd need to do something more involved?

Reply:
Updated the comment. Just add some nice wrapper for the generated objects section/layer/whatever to RDS.